src/base/bittorrent/torrentimpl.cpp

```diff
@@ -441,19 +477,14 @@ QVector<TrackerEntry> TorrentImpl::trackers() const
 
     for (const lt::announce_entry &tracker : nativeTrackers)
 #if (LIBTORRENT_VERSION_NUM >= 20000)
-        entries << fromNativeAnnouncerEntry(tracker, m_nativeHandle.info_hashes());
+        entries << fromNativeAnnouncerEntry(tracker, m_nativeHandle.info_hashes(), m_trackerPeerCounts[QString::fromStdString(tracker.url)]);
```

Comment:
if you don't mind, please extract `QString::fromStdString(tracker.url)` to a variable.